includes/data-port/class-sensei-import.php

```diff
@@ -43,7 +43,7 @@ function() {
 			add_action(
 				'admin_print_styles',
 				function() {
-					Sensei()->assets->enqueue( 'sensei-import', 'data-port/style.css', [ 'wp-components' ] );
+					Sensei()->assets->enqueue( 'sensei-import', 'data-port/style.css', [ 'sensei-wc-components', 'sensei-wp-components' ] );
```

Comment:
With this we can remove some current importer CSS (green button stylings), but it also needs the `sensei-color` class added to the `body` element. Probably better to do it in a separate PR though.